### PR TITLE
Additional Resources Page - RSC-118

### PR DESCRIPTION
**Pull request by rpokorny**
https://issues.sonatype.org/browse/RSC-118

image

### CODE REVIEW

File: gallery/src/pages/AdditionalResources.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        <a rel="noreferrer" href="https://reactjs.org/docs/thinking-in-react.html" target="_blank">Thinking in React</a>
+      </li>
+      <li className="nx-list__item">
+        IQ (and many other apps utilizing React and Redux together) heavily utilize the official{' '}
+        <a rel="noreferrer" href="https://react-redux.js.org/using-react-redux/connect-mapstate" target="_blank">
+          react-redux
+        </a>
+        {' '}connectors, as well as the{' '}
+        <a rel="noreferrer" href="https://redux-toolkit.js.org/introduction/quick-start" target="_blank">
+          Redux toolkit
```
Comment (rpokorny) [inline gallery/src/pages/AdditionalResources.tsx:50]: I had never heard of the redux "toolkit" before, so I think it is inaccurate to say it is used within IQ.

File: gallery/src/pages/AdditionalResources.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        </a>
+        , which is a set of patterns to reduce boilerplate in “vanilla Redux”.
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://www.youtube.com/watch?time_continue=781&v=nYkdrAPrdcw&feature=emb_logo"
```
Comment (rpokorny) [inline gallery/src/pages/AdditionalResources.tsx:56]: Did you intend for this to link to a point partway through the video?

**Comment by glenhunter** [comment]
I found this course quite helpful: https://reactforbeginners.com

File: gallery/src/pages/AdditionalResources.tsx (hunk 1 above)
Comment (timlevett) [reply gallery/src/pages/AdditionalResources.tsx:74]: fixed; 3ca289d

**Comment by timlevett** [comment]
> I found this course quite helpful: https://reactforbeginners.com

added.

File: gallery/src/pages/AdditionalResources.tsx (hunk 2 above)
Comment (timlevett) [reply gallery/src/pages/AdditionalResources.tsx]: That was the way it was shared with me. I reviewed the first bit of the video, and I think the time_continue helps get to the meat of the content.

File: gallery/src/pages/AdditionalResources.tsx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      </li>
+    </ul>
+
+    <h3 className="nx-h3 nx-tile__section-header">Getting your hands dirty</h3>
+    <ul className="nx-list nx-list--bulleted">
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://github.com/sonatype/sonatype-application-builder"
+           target="_blank">
+             Sonatype Application Builder (SAB) Template
```
Comment (glenhunter) [inline gallery/src/pages/AdditionalResources.tsx:95]: Maybe add a note that this is an internal Sonatype only project?

Comment (timlevett) [reply gallery/src/pages/AdditionalResources.tsx]: done in 18a0d7d

File: gallery/src/pages/AdditionalResources.tsx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+           target="_blank">
+             Internal Sonatype Application Builder (SAB) Template
+        </a>
+        {' - '}A great place to start for new projects and a fun place to poke around.
+        It is also setup as a GitHub Template, just click the "use this template" button (or click{' '}
+        <a rel="noreferrer"
+           href="https://github.com/sonatype/sonatype-application-builder/generate"
+           target="_blank">here
+        </a>
+        ) in GitHub to get started. Note this is an internal to Sonatype template.
```
Comment (glenhunter) [inline gallery/src/pages/AdditionalResources.tsx:103]:
```suggestion
        ) in GitHub to get started. Note that, at this time, this is an internal to Sonatype template.
```

File: gallery/src/pages/AdditionalResources.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+      </li>
+    </ul>
+
+    <h3 className="nx-h3 nx-tile__section-header">Good Reads</h3>
+    <ul className="nx-list nx-list--bulleted">
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://reactjs.org/docs/hello-world.html"
+           target="_blank">Main Concepts in React
+        </a>
```
Comment (rpokorny) [inline gallery/src/pages/AdditionalResources.tsx:62]:
```suggestion
        <a rel="noreferrer" href="https://reactjs.org/docs/hello-world.html" target="_blank">
          Main Concepts in React
        </a>
```

File: gallery/src/pages/AdditionalResources.tsx [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+      </li>
+    </ul>
+
+    <h3 className="nx-h3 nx-tile__section-header">Getting your hands dirty</h3>
+    <ul className="nx-list nx-list--bulleted">
+      <li className="nx-list__item">
+        <a rel="noreferrer"
+           href="https://github.com/sonatype/sonatype-application-builder"
+           target="_blank">
+             Internal Sonatype Application Builder (SAB) Template
```
Comment (rpokorny) [inline gallery/src/pages/AdditionalResources.tsx:95]:
```suggestion
          Internal Sonatype Application Builder (SAB) Template
```

File: gallery/src/pages/AdditionalResources.tsx [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+        </a>
+        {' - '}A great place to start for new projects and a fun place to poke around.
+        It is also setup as a GitHub Template, just click the "use this template" button (or click{' '}
+        <a rel="noreferrer"
+           href="https://github.com/sonatype/sonatype-application-builder/generate"
+           target="_blank">here
```
Comment (rpokorny) [inline gallery/src/pages/AdditionalResources.tsx:101]:
```suggestion
           target="_blank">
          here
```

**Comment by rpokorny** [comment]
I can't +1 this since I'm technically the one who opened it

**Pull request merged**